tests/components/awair/const.py

```diff
@@ -2,14 +2,17 @@
 
 import json
 
-from homeassistant.const import CONF_ACCESS_TOKEN
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_HOST
 
 from tests.common import load_fixture
 
 AWAIR_UUID = "awair_24947"
-CONFIG = {CONF_ACCESS_TOKEN: "12345"}
-UNIQUE_ID = "foo@bar.com"
-DEVICES_FIXTURE = json.loads(load_fixture("awair/devices.json"))
+CLOUD_CONFIG = {CONF_ACCESS_TOKEN: "12345"}
+LOCAL_CONFIG = {CONF_HOST: "192.0.2.5"}
+CLOUD_UNIQUE_ID = "foo@bar.com"
+LOCAL_UNIQUE_ID = "00:B0:D0:63:C2:26"
+CLOUD_DEVICES_FIXTURE = json.loads(load_fixture("awair/cloud_devices.json"))
```

Comment:
let's not load a fixture at the top level. Instead put it in a fixture so it's only loaded when a test that uses it is run. Scope=session makes sure it is only loaded once every test run:

https://github.com/home-assistant/core/blob/4a938ec33ee8df17e619fc64a519ef7c2ff856d8/tests/components/zwave_js/conftest.py#L226-L229